### PR TITLE
graphql conf 2025 landing page

### PR DESCRIPTION
**Pull request by YassinEldeeb**

### CODE REVIEW

File: src/app/conf/_components/grid-button.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          <div
+            key={index}
+            className="border border-white/10"
+            style={{
+              width: "60px",
+              height: "60px",
+            }}
+          ></div>
+        ))}
+        <a href={disabled ? undefined : href} className="relative z-0 col-span-8 row-span-2 text-white text-3xl font-semibold bg-[#E10098] hover:bg-[#ef00a3] flex items-center justify-center gap-2"
```
Comment (dimaMachina) [inline src/app/conf/_components/grid-button.tsx]: `<a>` without href is not good for SEO

better to use `const ComponentToUse = disabled ? 'span' : 'a'`

File: src/app/conf/2025/sponsors.tsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            <List
+              items={sponsorGold}
+              className="flex"
+              linkClassName="p-8 lg:py-14 h-28 lg:h-[170px]"
+            />
+          </>
+        )}
+        {sponsorSilver.length && (
+          <>
+            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-[rgba(255,255,255,0.4)]">
```
Comment (dimaMachina) [inline src/app/conf/2025/sponsors.tsx]:
```suggestion
            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-white/40">
```

File: src/app/conf/2025/sponsors.tsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+            <List
+              items={sponsorPlatinum}
+              className="flex"
+              linkClassName="p-8 lg:py-14 h-28 lg:h-[210px]"
+            />{" "}
+          </>
+        )}
+        {sponsorGold.length && (
+          <>
+            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-[rgba(255,255,255,0.4)]">
```
Comment (dimaMachina) [inline src/app/conf/2025/sponsors.tsx]:
```suggestion
            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-white/40">
```

File: src/app/conf/2025/sponsors.tsx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            <List
+              items={sponsorDiamond}
+              className="flex"
+              linkClassName="p-8 lg:p-16 h-28 lg:h-[220px]"
+            />
+          </>
+        )}
+        {sponsorPlatinum.length && (
+          <>
+            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-[rgba(255,255,255,0.4)]">
```
Comment (dimaMachina) [inline src/app/conf/2025/sponsors.tsx]:
```suggestion
            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-white/40">
```

File: src/app/conf/2025/sponsors.tsx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+      className="bg-conf-black"
+      style={{
+        fontFamily: rubik.style.fontFamily,
+      }}
+    >
+      <div className="container conf-block">
+        <h1 className={classes.title}>Thanks to our 2024 sponsors!</h1>
+        {sponsorDiamond.length && (
+          <>
+            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-[rgba(255,255,255,0.4)]">
```
Comment (dimaMachina) [inline src/app/conf/2025/sponsors.tsx]:
```suggestion
            <div className="flex items-center gap-2 mb-2 border-b-2 border-dotted pb-1.5 border-white/40">
```

File: src/app/conf/2025/page.tsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        <div
+          className="absolute bottom-0 left-1/2 -translate-x-1/2 w-[90%] h-px bg-white/10"
+          aria-hidden="true"
+        ></div>
+      </div>
+
+      <div className="container flex gap-20 flex-col pt-24">
+        <div className="flex gap-12 lg:gap-24 max-md:flex-col">
+          <div className="flex flex-col gap-5 text-white flex-1">
+            <h2 className="text-3xl lg:text-[45px]/[4rem] font-[400]">
```
Comment (dimaMachina) [inline src/app/conf/2025/page.tsx]:
```suggestion
            <h2 className="text-3xl lg:text-[45px]/[4rem] font-normal">
```

File: src/app/conf/2025/page.tsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+    // And swap it with the current element.
+    ;[array[currentIndex], array[randomIndex]] = [
+      array[randomIndex],
+      array[currentIndex],
+    ]
+  }
+
+  return array
+}
```
Comment (dimaMachina) [inline src/app/conf/2025/page.tsx]: this fn seems already exist? and can be resused

File: package.json [hunk 8]
```diff
@@ -34,6 +34,7 @@
     "hast-util-to-string": "3.0.1",
     "iframe-resizer-react": "^1.1.0",
     "leaflet": "^1.9.4",
+    "lucide-react": "^0.468.0",
```
Comment (saihaj) [inline package.json:37]: forgive me; we have too many icon packages. Will figure out cleaning all of those....

**Review by saihaj — APPROVED** [review]

**Pull request merged**